docs/user-guide/arrays.rst

```diff
@@ -0,0 +1,487 @@
+
+Working With Arrays
+===================
+
+Creating an array
+-----------------
+
+Zarr has several functions for creating arrays. For example:
+
+.. ipython:: python
+
+   import zarr
+
+   z = zarr.zeros((10000, 10000), chunks=(1000, 1000), dtype='i4')
```

Comment:
```suggestion
   z = zarr.open((10000, 10000), chunks=(1000, 1000), dtype='i4')
```

I don't think the first example on this page should initialize the data at creation. The next section talks about writing values, so simply `open` or whatever.

Reply:
Also I think the "persistent storage" bit should be moved up here. Most new users are looking for a way to store data on disk, starting off with MemoryStore is confusing (from my personal experience)

Reply:
I agree. The main point of using zarr over numpy is persistence